casper/src/main/resources/PoS.rhox

```diff
@@ -108,7 +146,9 @@ new  PoS,
             } |
 
             contract PoS(@"closeBlock", ackCh) = {
-              new blockDataCh, getBlockData(`rho:block:data`), stdout(`rho:io:stdout`) in {
+              new blockDataCh,
+                  getBlockData(`rho:block:data`),
+                  stdout(`rho:io:stdout`) in {
```

Comment:
stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout stdout

Reply:
This one is intended: https://rchain.atlassian.net/browse/RCHAIN-3462